packages/data-context/src/data/ProjectLifecycleManager.ts

```diff
@@ -103,7 +103,7 @@ export class ProjectLifecycleManager {
 
   async getProjectId (): Promise<string | null> {
     try {
-      const contents = await this.getConfigFileContents()
+      const contents = await this.ctx.project.getConfig()
```

Comment:
@tgriesser is this something that https://github.com/cypress-io/tech-briefs/pull/19/files will help with? Maybe there could be better guards around calling `this.getConfigFileContents` - outside of config file resolution, there shouldn't be any need for code to be reading the config file in isolation, which was the source of this bug.

Reply:
Yeah I think it might help. I think there was an old `read` helper used by old integration tests that used `cypress.json` which read the config file in isolation, if we can rip that out we can make `getConfigFileContents ` private.